packages/server/graphql/types/Poll.ts

```diff
@@ -12,6 +13,13 @@ const Poll = new GraphQLObjectType<any, GQLContext>({
   isTypeOf: ({title}) => !!title,
   fields: () => ({
     ...(threadableFields() as any),
+    createdByUser: {
+      type: GraphQLNonNull(User),
+      description: 'The user that created the item',
+      resolve: ({createdById}, _args, {dataLoader}: GQLContext) => {
+        return dataLoader.get('users').load(createdById)
+      }
+    },
```

Comment:
I added it here as a non-nullable field, Polls can't be created without a user. Threadable fields are providing nullable type

Reply:
Good point. I can remove it. The question is, this PR was closed and not merged. @tiffanyhan are we still doing it or was it dropped for some reason? With polls, this is also a matter of changing the DB schema as currently, we have 
```
      "createdById" VARCHAR(100) NOT NULL REFERENCES "User"("id"),
```
so it'd have to be changed to a nullable field.